### ISSUE TITLE
missing `permission.display_role_in_exception`?

### ISSUE THREAD

**Issue by Yoeriwalstra** [spatie/laravel-permission#1467]
Hi,

tl;dr: `UnauthorizedException::forRolesOrPermissions` checks for non-existent configuration key `permission.display_role_in_exception`.

I'm updating this package from version `1.18` to the latest version in one of my company's projects. I downloaded the new `permission.php` file (since `v1.18` still uses the old `laravel-permission.php` file) and was comparing the old and new config files, to see if there were any custom values I needed to copy. I also went through the (source) code to check for the usages of the config values. 
In the `Spatie\Permission\Exceptions\UnauthorizedException` class I noticed that the `forRolesOrPermissions` method contained a condition that checked if two config settings were `true`:
```
if (config('permission.display_permission_in_exception') && config('permission.display_role_in_exception'))
```
I found the first config key in `permission.php`, but not the second. Did i stumble upon a bug/missing config key, or am I missing something here?

Kind regards,

Yoeri

**Comment by drbyte** [comment]
@Yoeriwalstra Good catch!

On the plus side at least it's falling back to the conservative response, which doesn't disclose anything unintentional. And technically there's little reason to ever set it to `true` other than when in a dev phase.

A PR to add it to the default config file with a default of `false` is welcome.

**Comment by Yoeriwalstra** [comment]
Gotcha. I hope this suffices: https://github.com/spatie/laravel-permission/pull/1468.

Haven't contributed to any open source package before, so I hope I got everything right (in line with your `CONTRIBUTING.md`).

Kind regards,

yoeri
